packages/base/src/Internal/Container.hs

```diff
@@ -40,9 +40,14 @@ import Prelude hiding ((<>))
 {- | Creates a real vector containing a range of values:
 
 >>> linspace 5 (-3,7::Double)
-fromList [-3.0,-0.5,2.0,4.5,7.0]@
+[-3.0,-0.5,2.0,4.5,7.0]
+it :: Vector Double
 
 >>> linspace 5 (8,2+i) :: Vector (Complex Double)
+
+<interactive>:6:17: ERROR:
```

Comment:
Why are these errors here?

Reply:
Due to my sloppiness, have already fixed them, going to push soon.

Initially I assumed that the examples in the original haddock comments were copied verbatim
from interactive sessions. That turned out not always the case. Some, for example these 
two you picked, must have been constructed manually since they contained bugs.
It took me a while to realize that, admittedly I am not very familiar with the package.

As I was replaying these examples one by one in ghci with my modified Show instances
when I encountered a problem and could not immediately figure out what was wrong I just
copied the error messages into the comment as a kind of bookmark. Then later I came back
to them to fix them. I forgot these two.